### PR TITLE
fix failing animation tests

### PR DESCRIPTION
**Pull request by harshil21**
telegram changed some animation return values so

### CODE REVIEW

File: tests/test_animation.py [hunk 1]
```diff
@@ -145,9 +145,11 @@ def test_send_animation_url_file(self, bot, chat_id, animation):
         assert message.animation.file_unique_id != ''
 
         assert message.animation.duration == animation.duration
-        assert message.animation.file_name == animation.file_name
+        assert message.animation.file_name.startswith(
+            'game.gif'
+        ) == animation.file_name.startswith('game.gif')
         assert message.animation.mime_type == animation.mime_type
-        assert message.animation.file_size == animation.file_size
+        assert message.animation.file_size == 4127 and animation.file_size == 5859
```
Comment (Bibo-Joshi) [inline tests/test_animation.py:152]: tbh, I'd be okay with just dropping the file size tests …

File: tests/test_animation.py [hunk 2]
```diff
@@ -145,9 +145,11 @@ def test_send_animation_url_file(self, bot, chat_id, animation):
         assert message.animation.file_unique_id != ''
 
         assert message.animation.duration == animation.duration
-        assert message.animation.file_name == animation.file_name
+        assert message.animation.file_name.startswith(
+            'game.gif'
+        ) == animation.file_name.startswith('game.gif')
```
Comment (Bibo-Joshi) [inline tests/test_animation.py:150]: what does  this do? do the file names differ?

Comment (harshil21) [reply tests/test_animation.py:150]: one is `game.gif.mp4` other one is `game.gif.webm`

Comment (Bibo-Joshi) [reply tests/test_animation.py:150]: wtf. what happens if you replace the bitly link with `'https://python-telegram-bot.org/static/testfiles/game.gif?no=cache'` ? (the parameter should avoid caching)

Comment (harshil21) [reply tests/test_animation.py:150]: same result

Comment (Bibo-Joshi) [reply tests/test_animation.py:150]: geee … okay let's keep it then. You can merge after removing the file size test.
btw deepsource promised to fix the failing tests soonish

**Pull request merged**